### DISCUSSION TITLE
Support of the n39 band

### DISCUSSION THREAD

**Discussion by MutuThambi** [srsran/srsRAN_Project#625] [General Help]
I have seen in the documentation that the n39 band is not currently supported by the srsRAN project. 
Is there a way to set up a gNB in the n39 band with the current version or is it planned to be included in future releases?

Thanks.

**Comment by carlo-gal** [comment]
Hi @MutuThambi,

n39 is actually supported, probably we missed that band in the documentation. You should be able to use it.
Thanks.
